src/http/server.rs

```diff
@@ -1130,12 +1199,14 @@ pub mod ws {
     }
 
     impl Sender for EspHttpWsConnection {
+        /// As per [`crate::http::EspHttpWsConnection::send`](EspHttpWsConnection::send)
```

Comment:
Ditto here and also for `Receiver` - not sure trait impls need documentation. It is another topic that the traits themselves need documentation, but this is for `embedded-svc` :)